packages/tablecloth-native/tablecloth-native.0.0.2/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis:
+  "Native OCaml library implementing Tablecloth, a cross-platform standard library for OCaml, Bucklescript and ReasonML"
+description: "Longer description"
+maintainer: "Paul Biggar <paul@darklang.com>"
+authors: "Paul Biggar <paul@darklang.com>"
+license: "MIT with some exceptions"
+homepage: "https://github.com/darklang/tablecloth"
+bug-reports: "https://github.com/darklang/tablecloth/issues"
+depends: [
+  "ocaml"
+  "ocamlfind"
+  "dune"
+  "base" {>= "v10.0"}
+  "str"
+]
+build: ["dune" "build" "-p" name]
```

Comment:
```suggestion
build: ["dune" "build" "-p" name "-j" jobs]
```
The idiomatic call to dune usually have also `"-j" jobs` so than people can parallelize (or not) as they wish

Reply:
👍